### PR TITLE
feat: add wallet connect v2 routes

### PR DESCRIPTION
**Pull request by reallybeard**
## Description

- This just plumbs the routes for @0xApotheosis to start working on the v2 version.

## Pull Request Type

- [ ] :bug: Bug fix (Non-breaking Change: Fixes an issue)
- [ ] :hammer_and_wrench: Chore (Non-breaking Change: Doc updates, pkg upgrades, typos, etc..)
- [x] :nail_care: New Feature (Breaking/Non-breaking Change)

## Issue (if applicable)

closes #

## Risk

No risk behind a feature flag

### Operations

No testing

## Screenshots (if applicable)
Screenshot 2023-07-19 at 10 14 39 AM
Screenshot 2023-07-19 at 10 14 42 AM
Screenshot 2023-07-19 at 10 14 47 AM

### CODE REVIEW

File: src/context/WalletProvider/WalletConnectV2/components/Create.tsx [hunk 1]
```diff
@@ -0,0 +1,7 @@
+export const WalletConnectV2Create = () => {
+  return (
+    <>
+      <p>Add new connection</p>
```
Comment (0xApotheosis) [inline src/context/WalletProvider/WalletConnectV2/components/Create.tsx:4]: Translation string?

File: src/assets/translations/en/main.json [hunk 2]
```diff
@@ -1445,6 +1445,22 @@
       "body": "By connecting a wallet, you agree to ShapeShift's Terms of Service and acknowledge that you have read and understand the ShapeShift disclaimer.",
       "footer": "Don't have a wallet?",
       "create": "Create one"
+    },
+    "walletConnectV2": {
+      "start": {
```
Comment (0xApotheosis) [inline src/assets/translations/en/main.json:1450]: Are we intentionally missing `body` here?

Screenshot 2023-07-21 at 7 54 01 am

**Review by 0xApotheosis — COMMENTED** [review]
Does what it says on the box, a few suggestions.

Maybe a placeholder here when there are no saved connections?

Screenshot 2023-07-21 at 7 54 53 am

File: src/assets/translations/en/main.json (hunk 2 above)
Comment (reallybeard) [reply src/assets/translations/en/main.json:1450]: I didn't know what to put there, so left it blank. We can remove.

**Review by 0xApotheosis — APPROVED** [review]

**Pull request merged**